samples/subsys/logging/logger/sample.yaml

```diff
@@ -5,6 +5,7 @@ sample:
 tests:
   sample.logger.basic:
     tags: logging
+    filter: not CONFIG_LOG_BACKEND_RB
```

Comment:
Can't we integrate the tooling with sanitycheck instead?  Whitelisting tests (OK, this is technicaly a sample) is a last resort to work around bugs or platform misfeatures that have no reasonable resolution.  There's no reason we can't add the host-side log dumper to the sanitycheck console integrations.